src/static/js/main.js

```diff
@@ -46,17 +46,58 @@ $(function () {
       contentType: 'application/json'
     }).done(function (response) {
       $(form).find('button[type=submit]').toggleClass('loading')
-      // This is used allow each template to implement its own response handler
-      // if (FORM_HANDLERS[$(form).attr('name')]) {
-      //   FORM_HANDLERS[$(form).attr('name')](form, response)
-      // }
+     
 
-      console.log('asdasdasd', THE_ORG_BOOK_APP_URL)
+      console.log('asdasdasd')
```

Comment:
Comment clean up - either just remove this logging or change it to what the logging demonstrates.